<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagChip</class>
 <widget class="QFrame" name="TagChip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="tag_label">
     <property name="text">
      <string>Tag Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_button">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>assets/icons/close.svg</normaloff>
       <normalon>assets/icons/close.svg</normalon>
       <disabledoff>assets/icons/close.svg</disabledoff>
       <disabledon>assets/icons/close.svg</disabledon>
       <activeoff>assets/icons/close.svg</activeoff>
       <activeon>assets/icons/close.svg</activeon>
       <selectedoff>assets/icons/close.svg</selectedoff>
       <selectedon>assets/icons/close.svg</selectedon>
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
